blosc/blosc2.c

```diff
@@ -886,6 +888,26 @@ uint8_t* pipeline_c(struct thread_context* thread_context, const int32_t bsize,
       case BLOSC_TRUNC_PREC:
         truncate_precision(filters_meta[i], typesize, bsize, _src, _dest);
         break;
+      case BLOSC_UDFILTER:
```

Comment:
Add a small comment like:

// Look for the user filter id in filters_meta and run it

Reply:
Fixed

Reply:
Fixed